graylog2-web-interface/src/views/components/searchbar/saved-search/SavedSearchControls.jsx

```diff
@@ -258,21 +257,21 @@ class SavedSearchControls extends React.Component<Props, State> {
             <ButtonGroup>
               <>
                 <Button title={title} ref={(elem) => { this.formTarget = elem; }} onClick={this.toggleFormModal}>
-                  <Icon style={{ color: savedSearchColor }} name={savedSearchStyle} /> Save
+                  <Icon style={{ color: savedSearchColor }} name="star" /> Save
```

Comment:
Do we need both a solid and outlined `star` here?

https://fontawesome.com/icons/star?style=regular
https://fontawesome.com/icons/star?style=solid

Reply:
Thanks again for the hint, I've added the ability to outline icons and updated the icons which had `-o` in their name.

I've changed the `Icon` props interface in the same step, I think it is now easier to understand, but it would be great to hear your opinion. There are no changes needed in our plugins.